jerry-core/parser/regexp/re-parser.cpp

```diff
@@ -403,11 +467,12 @@ re_parse_char_class (re_parser_ctx_t *parser_ctx_p, /**< number of classes */
         append_char_class (re_ctx_p, 0x2030UL, 0x205EUL);
         append_char_class (re_ctx_p, 0x2060UL, 0x2FFFUL);
         append_char_class (re_ctx_p, 0x3001UL, 0xFEFEUL);
```

Comment:
@ruben-ayrapetyan, you are right. I can change some of these that exists in `lit-char-helpers.h`. What about the missing ones? Where should I define them? In the `lit-char-helpers.h` or re-`parser.cpp`?